docs/CHANGELOG.rst

```diff
@@ -12,12 +12,17 @@ Unreleased
 
 Changed
 -------
+- **BACKWARD INCOMPATIBLE:** Refactor ``trimmomatic-single`` and ``trimmomatic-paired``
+  processes
 - Bump STAR aligner version in ``resolwebio/rnaseq`` docker image to 2.5.4b
 - Bump Primerclip version in ``resolwebio/dnaseq`` docker image
 
 Added
 -----
 - Add CNVKit, LoFreq and GATK to ``resolwebio/dnaseq`` docker image
+- **BACKWARD INCOMPATIBLE:** Merge ``align-bwa-trim`` and ``align-bwa-trim2``
```

Comment:
This that are backward incompatible are **changes** by definition, not additions.

Also, backward incompatible changes should be at the top of each CHANGELOG section.